src/storybook/stand-alone-documentaion/typography/text-colors/text-colors.jsx

```diff
@@ -0,0 +1,24 @@
+import { ColorDescription } from "../color-description/color-description";
+import { Frame } from "../../../components";
+
+export const TextColors = () => {
+  return (
+    <Frame>
+      <ColorDescription colorName="primary-text-color" description="Use for default text color" />
```

Comment:
use a map/array for these.

We will need something like this for the colors maybe a component that accepts the map/array so we could manage all of them in one place

Reply:
I did not implement this like this because evigny once said he want to change the colors grid appearance.... even so Its not looks like it will happened until release